### PR TITLE
Add more cross references and update Situation Lead terminology

### PR DESCRIPTION
**Pull request by pauldoomgov**

### CODE REVIEW

**Review by sharms — APPROVED** [review]
LGTM

File: _articles/secops-incident-response-guide.md [hunk 1]
```diff
@@ -32,25 +33,25 @@ At a high level, incident response follows this process:
 
 An TTS staff member inside or outside the login.gov team (the reporter) notices and reports a login.gov-related incident, using the TTS incident response process and notifying the login.gov team in the #login-situation Slack channel.
 
-- The first responder on the login.gov team (which could be the reporter if the reporter is on the team) becomes the initial Incident Commander (IC).
+- The first responder on the login.gov team (which could be the reporter if the reporter is on the team) becomes the initial Situation Lead (IC).
```
Comment (zachmargolis) [inline _articles/secops-incident-response-guide.md:36]: should we update the abbreviation at the end of the line to match?

```suggestion
- The first responder on the login.gov team (which could be the reporter if the reporter is on the team) becomes the initial Situation Lead (SL).
```

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/secops-incident-response-guide.md (hunk 1 above)
Comment (pauldoomgov) [reply _articles/secops-incident-response-guide.md]: YES!   My lazy s/// missed a bunch of cases.  Thanks for being thorough!

**Pull request merged**